plugin/evm/vm.go

```diff
@@ -1745,11 +1745,15 @@ func (vm *VM) chainConfigExtra() *extras.ChainConfig {
 	return params.GetExtra(vm.chainConfig)
 }
 
+func (vm *VM) rulesExtra(number *big.Int, time uint64) extras.Rules {
```

Comment:
(optional) Since you're calling the other one `currentRules()`, consider just calling this `rules()`.

Reply:
Done!